networkit/cpp/randomization/Curveball.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Curveball.cpp
+ *
+ *  Created on: 26.05.2018
+ *      Author:  Hung Tran <htran@ae.cs.uni-frankfurt.de>, Manuel Penschuck <networkit@manuel.jetzt>
+ */
+
+#include "../auxiliary/Random.h"
+
+#include "Curveball.h"
+#include "CurveballImpl.h"
+
+namespace NetworKit {
+
+Curveball::Curveball(const NetworKit::Graph &G) :
+    impl_(new CurveballDetails::CurveballIM{G})
```

Comment:
Existing NetworKit code uses camelCase even for internal variables. While I can see the point of the underscore, consistency should be preferred here. Can you change the internal variables to use camelCase?